pkg/apis/apis.go

```diff
@@ -0,0 +1,32 @@
+/*
```

Comment:
Could we name the package to api? I think it is a convension in k8s community. WDYT

Reply:
The kubebuilder automatically generate apis dir. I will fix it.